Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -5211,6 +5211,63 @@ def get_machine_action_command(client, args):
     )
 
 
+def fetch_indicators_command(client, args):
+
+    i = 1
```

Comment:
Please give the variable a more meaningful name than _i_ and in the function `get_machines` it sets the page num to zero by default `page_num = 0 if not page_num else (page_num - 1)` shouldn't it start at zero here as well ? If it starts with 1 the first skip will be 10000